cmd/nerdctl/image/image_list_test.go

```diff
@@ -144,7 +145,9 @@ LABEL foo=bar
 LABEL version=0.1
 RUN echo "actually creating a layer so that docker sets the createdAt time"
 `, testutil.CommonImage)
-			buildCtx := testhelpers.CreateBuildContext(t, dockerfile)
+			buildCtx := data.TempDir()
+			err := os.WriteFile(filepath.Join(buildCtx, "Dockerfile"), []byte(dockerfile), 0o600)
+			assert.NilError(helpers.T(), err)
```

Comment:
why we cannot use the helper directly?

Reply:
(I say "copy it over" because a bunch of legacy tests are still dependent on the existing helper and would not be able to use the new one)